service/persistentvolume/framework.go

```diff
@@ -0,0 +1,114 @@
+package persistentvolume
+
+import (
+	"fmt"
+	"sync"
+
+	gsclient "github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/informer"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v1"
```

Comment:
This import looks wrong.

Reply:
Thank you <3

Reply:
`dep ensure` removed that garbage. Thanks you saved me from many struggles :)